### PR TITLE
[plugin.video.repubblicaitaliana] v1.0.0

### PR DESCRIPTION
**Pull request by Testato**
### Description
RepubblicaItaliana Kodi plugin
Italian politics - TV and Radio channels
### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.repubblicaitaliana/addon.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.repubblicaitaliana" name="RepubblicaItaliana" version="1.0.0" provider-name="Testato">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+    <import addon="inputstream.adaptive" version="2.6.7"/>
+    <import addon="plugin.video.youtube" version="6.8.10+matrix.1"/>
```
Comment (enen92) [inline plugin.video.repubblicaitaliana/addon.xml:6]: You might want to rely on tubed for matrix in the future, it's the new version of YouTube for kodi

**Review by enen92 — APPROVED** [review]
Looks clean

File: plugin.video.repubblicaitaliana/addon.xml (hunk 1 above)
Comment (Testato) [reply plugin.video.repubblicaitaliana/addon.xml:6]: How i can swap from youtube to tubed ?

Now i call the video in this way:
link = 'plugin://plugin.video.youtube/play/?video_id=_pjPv7dS-_w'

I tried:
link = 'plugin://plugin.video.tubed/play/?video_id=_pjPv7dS-_w'

But it do not work.
Sorry if I ask here but I asked on Kodi forum without answer

Thanks

Comment (enen92) [reply plugin.video.repubblicaitaliana/addon.xml:6]: See here:
https://github.com/XBMC-Addons/plugin.video.nasa/commit/169032edcdf71ffda247ad2a605824df7018d25b

**Pull request merged**